packages/plugin-constraints/sources/types/tau-prolog.d.ts

```diff
@@ -0,0 +1,275 @@
+declare module 'tau-prolog' {
```

Comment:
I could open a ticket on their side yeah :) They can add a d.ts file and point to it in the `typings` property in their `package.json`. It should work out of the box.

That will take a bit longer than committing this file in the repo though, because these types are neither complete nor (probably) completely correct.

Reply:
I mean, it's still strictly better than no types at all, after all 😛

Reply:
Already noticed a mistake though... oh well